css/src/tokens/colors.scss

```diff
@@ -264,8 +267,10 @@ $gradients: (
 	'purple-blue': (
 		$gradient-text-purple,
 		$gradient-text-blue
+	),
+	'ask-learn': (
```

Comment:
Can you please avoid feature specific names and do something more generic? `yellow-purple`?

Reply:
renamed to red-purple